pkg/cloud/vsphere/context/cluster_context.go

```diff
@@ -186,6 +187,16 @@ func (c *ClusterContext) Pass() string {
 	return c.pass
 }
 
+// GetSSH returns a SSH client that can be used to execute commands against the vSphere endpoint
+func (c *ClusterContext) GetSSH() *ssh.SSH {
```

Comment:
Hi @moshloop,

What's the reason this needs to be part of the Cluster context? The `user` and `pass`? I think I'd rather defer this to a secret if possible.